src/jobs/deploy_job.yml

```diff
@@ -53,6 +53,9 @@ parameters:
   slack_fail_branches:
     type: string
     default: master
+  working_directory:
```

Comment:
Jobs already have working_directory parameter. See: https://circleci.com/docs/2.0/configuration-reference/#jobs
Rename it to void conflict.